README.md

```diff
@@ -50,12 +54,14 @@ the previews for each theme in the [section](#previews) below or in this other
     ```
     include ./theme.conf
     ```
-    
+
+### Conda
+
 If you using the ``conda`` package manager, you may also install these themes
 with the following command:
 
     ```bash
```

Comment:
This block of code is not being rendered correctly. I think the problem is with the indentation, try to remove it.

![Screenshot 2019-06-29 09:56:08](https://user-images.githubusercontent.com/7031675/60381574-44e1ed00-9a57-11e9-93b8-ef6fb0b24179.png)